spec/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_spec.rb

```diff
@@ -1,5 +1,9 @@
 require 'support/ansible_shared/automation_manager/configuration_script'
 
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript do
+  let(:provider_with_authentication)       { FactoryGirl.create(:provider_embedded_ansible, :with_authentication) }
+  let(:manager_with_authentication)        { provider_with_authentication.managers.first }
+  let(:manager_with_configuration_scripts) { FactoryGirl.create(:embedded_automation_manager_ansible, :provider, :configuration_script) }
+
   it_behaves_like 'ansible configuration_script'
```

Comment:
I tried, but could not make them parameters. Seems parameters can be only primitives not variables defined by a `let` block.